scripts/staging/SystemML-NN/nn/layers/dropout.dml

```diff
@@ -42,10 +42,7 @@ forward = function(matrix[double] X, double p, int seed)
    *  - out: Ouptuts, of same shape as X.
    *  - mask: Dropout mask used to compute the output.
    */
-  if (seed == -1) {
-    seed = as.integer(floor(as.scalar(rand(rows=1, cols=1, min=1, max=100000))))
-  }
```

Comment:
These three lines need to be added back so that a user can indicate that the dropout layer should use a random seed.